File: drawable-animation/app/src/main/java/com/cglotr/demo/MainActivity.java
package com.cglotr.demo;

import android.graphics.drawable.AnimationDrawable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    private AnimationDrawable wifiAnimation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ImageView imageView = findViewById(R.id.image);
        imageView.setBackgroundResource(R.drawable.animation);

        wifiAnimation = (AnimationDrawable) imageView.getBackground();
    }

    @Override
    public void onWindowFocusChanged(boolean hasFocus) {
        super.onWindowFocusChanged(hasFocus);

        wifiAnimation.start();
    }
}

File: list-view/app/src/main/java/com/example/cglotr/listv/MainActivity.kt
package com.example.cglotr.listv

import android.app.Activity
import android.os.Bundle
import android.widget.ListView
import com.example.cglotr.listv.ui.ListViewAdapter

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val listView = findViewById<ListView>(R.id.main_listView)
        listView.adapter = ListViewAdapter(this)
    }
}
